grid/chat/delete-message/profile.supr

```diff
@@ -0,0 +1,121 @@
+"""
```

Comment:
I don't know from where this comes, but adding new use-cases in PoC PR for recording validation seems incorrect.

Reply:
Why isn't correct to have one working example in proof of concept PR?

Reply:
It is fine, but this looks as added use-case or at least refactored, which would be better review separately.

Reply:
True, I moved it to #283 and I'll revert changes in this PR. For context: 

#263 - contains setup for continuous testing without new sdk
- #283 - contains prepared chat/messages (and also chat/delete-message)
- #280 - contains prepared other usecases under chat scope (I still need to update tests and snapshots)